features/step_definitions/is.rb

```diff
@@ -46,3 +46,27 @@
     raise "ImageStreamTag #{istag_name} never created"
   end
 end
+
+Given /^the #{QUOTED} #{QUOTED} CRD was finalized$/ do |crd_type, crd_name|
```

Comment:
what does it mean `finalized`? Could you submit scenario where this step is used so it is more obvious whether the step name is meaningful?

Reply:
There are other possible simplifications but it is hard to suggest them now before usage of the step becomes clearer.

Reply:
I want a scenario that the resource could be recreated after destroy.